src/main/java/com/microsoftopentechnologies/windowsazurestorage/AzureBlobAction.java

```diff
@@ -1,34 +1,27 @@
 package com.microsoftopentechnologies.windowsazurestorage;
 
+import com.microsoftopentechnologies.windowsazurestorage.WAStoragePublisher.WAStorageDescriptor;
+import com.microsoftopentechnologies.windowsazurestorage.beans.StorageAccountInfo;
+import hudson.model.Run;
+import hudson.model.RunAction;
 import java.io.IOException;
 import java.util.List;
-
 import javax.servlet.ServletException;
-
 import jenkins.model.Jenkins;
-
 import org.acegisecurity.Authentication;
 import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.StaplerResponse;
 
-import hudson.model.AbstractBuild;
-import hudson.model.RunAction;
-import hudson.model.Run;
-
-import com.microsoftopentechnologies.windowsazurestorage.WAStoragePublisher;
-import com.microsoftopentechnologies.windowsazurestorage.WAStoragePublisher.WAStorageDescriptor;
-import com.microsoftopentechnologies.windowsazurestorage.beans.StorageAccountInfo;
-
 
 public class AzureBlobAction implements RunAction {
-	private final AbstractBuild build;
+	private final Run build;
```

Comment:
This should actually be removed altogether.  The inclusion of this field causes GC issues, since the Runcannot be unloaded by the GC while the pointer is still live.